packages/salesforcedx-vscode-apex/package.nls.json

```diff
@@ -10,10 +10,11 @@
   "java_home_description": "Specifies the folder path to the Java 8 or Java 11 runtime used to launch the Apex Language Server (for example, /Library/Java/JavaVirtualMachines/jdk1.8.0_131.jdk/Contents/Home).",
   "java_memory_description": "Specifies the amount of memory allocation to the Apex Language Server in MB or null to use the default value.",
   "apex_semantic_errors_description": "Allow Apex Language Server to surface semantic errors",
+  "apex_completion_statistics_description": "List statistics for Apex Completions",
```

Comment:
@brpowell I could not find a way to have this be hidden from the settings UI. Do you happen to have more info on this?